docs/siem/detections/alerts-ui-manage.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[alerts-ui-manage]]
+[role="xpack"]
+== Managing detection alerts
+
+The Detections page displays all alerts. From the Alerts table, you can change
+an alert's status, and start investigating and analyzing alerts in Timeline.
+
+TIP: From Timeline, you can <<cases-ui-open, create cases>> to track issues and
+share information with colleagues.
+
+To view detection alerts created by a specific rule, you can:
+
+* Filter for a specific rule in the KQL bar (for example,
+`signal.rule.name :"SSH (Secure Shell) from the Internet"`).
+* View detection alerts in the *Rule details* page (click
+*Manage detection rules* -> rule name in the *All rules* table).
+
+NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the 
+*Detections* and *Rule details* pages, and in Timeline when either `All` or
+`Detection alerts` is selected. 
+
+TIP: Use the icons in the upper left corner of the Alerts table to customize
+displayed columns and row renderers, and view the table in full screen mode.
+
+[float]
+[[detection-alert-status]]
+=== Change alert statuses
+
+You can set an alert's status to indicate whether it needs to be investigated
+(`Open`), is under active investigation (`In progress`), or resolved
+(`Closed`). By default, the Alerts table displays open alerts. To view alerts
+with other statuses, click *In progress* or *Closed*.
+
+To change alert statuses, either:
+
+* In the alert's row, click the *more options* icon, and then select the
+required status (*Mark in progress*, *Close alert*, or *Open alert*).
+* In the Alerts table, select all the alerts you want to change, and then select
+*Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
+
+[float]
+[[signals-to-timelines]]
+=== Send alerts to Timeline
+
+To send an alert in Timeline, click the *Investigate in timeline* icon.
+
+If the rule that generated the alert uses a Timeline template, when you
+investigate the alert in Timeline, the dropzone query values defined in the
+template are replaced with their corresponding alert values.
+
+// * `host.name`
+// * `host.hostname`
+// * `host.domain`
+// * `host.id`
+// * `host.ip`
+// * `client.ip`
+// * `destination.ip`
+// * `server.ip`
+// * `source.ip`
+// * `network.community_id`
+// * `user.name`
+// * `process.name`
```

Comment:
Are these comments placeholders for something, or can they be removed?

Reply:
I think they can be removed but I need to check with @XavierM. Worst case, I'll remove them in a separate PR.